services/director-v2/tests/unit/test_utils_osparc_variables.py

```diff
@@ -223,8 +248,8 @@ async def test_substitute_vendor_secrets_in_specs(
     mock_get_vendor_secrets: None, fake_app: FastAPI, faker: Faker
 ):
     specs = {
-        "api_key": "${OSPARC_VARIABLE_VENDOR_SECRET_ONE}",
-        "api_secret": "${OSPARC_VARIABLE_VENDOR_SECRET_TWO}",
+        "vendor_secret_one": "${OSPARC_VARIABLE_VENDOR_SECRET_ONE}",
+        "vendor_secret_two": "${OSPARC_VARIABLE_VENDOR_SECRET_TWO}",
```

Comment:
Q: why was this renamed?

Reply:
AFAIK it does not have any effect. I just wanted to align the names ...